### DISCUSSION TITLE
A DaoClean method would be nice

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#1421] [Q&A]
Without using a helper it is very messy to create a new clean Dao.  Could you add this to core.App?

```golang
func (app core.App) DaoClean() *daos.Dao {
	return daos.New(app.Dao().DB())
}
```

**Comment by ganigeorgiev** [comment]
I don't think it is necessary and will just clutter further the app interface.

I think the daos factory is clear and simple enough and there is no need for helpers (_also please note that if you want more control on the db ops flow, there is also `daos.NewMultiDB()`_).
